docker/apply_ci.dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM cimg/ruby:3.2.2-browsers
 MAINTAINER apply for legal aid team
 
-RUN wget https://mirrors.kernel.org/ubuntu/pool/main/libf/libffi/libffi6_3.2.1-8_amd64.deb -O /tmp/libffi6_3.2.1-8_amd64.deb
+RUN curl --proto "=https" https://mirrors.kernel.org/ubuntu/pool/main/libf/libffi/libffi6_3.2.1-8_amd64.deb -o /tmp/libffi6_3.2.1-8_amd64.deb
```

Comment:
This will only get built on a timer each morning, so have you tested this locally @naseberry?
After the last apply_ci.dockerfile change issue 😄

Reply:
reverted, cURL redirect to an error page as source only accepts http requests